config-model-api/src/main/java/com/yahoo/config/application/api/DeploymentInstanceSpec.java

```diff
@@ -182,9 +155,10 @@ public boolean canChangeRevisionAt(Instant instant) {
     }
 
     /** Returns the athenz domain if configured */
-    public Optional<AthenzDomain> athenzDomain() { return athenzDomain; }
+    // TODO jonmv: Remove when the commit with this comment is deployed on all config servers.
```

Comment:
That's not sufficient because it is accessed from the config model. Better to give a date where you can be sure all config models accessing it are gone, without requiring the random person seeing this in the future to do detective work, e.g "After February 2020".

Reply:
Of course. I have no idea what I was thinking :)

Reply:
```suggestion
    // TODO jonmv: Remove when 7.162 is older than the oldest deployed version.
```